.travis.yml

```diff
@@ -2,9 +2,6 @@ language: python
 python:
 - 2.7
 - 3.5
-
-sudo: "required"
```

Comment:
I'm removing this line, the default is false and container based infrastructure.